src/g3log/loglevels.hpp

```diff
@@ -115,8 +115,9 @@ namespace g3 {
    namespace only_change_at_initialization {
       // Enable/Disable a log level {DEBUG,INFO,WARNING,FATAL}
       void setLogLevel(LEVELS level, bool enabled_status);
+	  void addLogLevel(LEVELS level, bool enabled_status = true);
       std::string printLevels();
-      void reset();
+      void reset(bool remove_custom_levels = false);
```

Comment:
please remove the boolean. I am a strong believer in SRP and this function has now two responsibilities.